rpc/documentation/api.md

```diff
@@ -419,6 +420,40 @@ usage and relevant transactions.
 
 - `FailedPrecondition`: The wallet or consensus RPC server has not been opened.
 
+**Stability:** Unstable
+___
+
+#### `SpvSync`
+
+The `SpvSync` method starts the syncing process for the --spv wallet.  It will
```

Comment:
the --spv command line option has nothing to do with this rpc, other than both create and run a syncer.